assets/js/lesson_5/hw_10.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * I will give names for variables like taskOne, taskTwo , etc.
+ * and add description to every new task
+ */
+
+// 1.Show numbers from 10 to 20 in one line
+
+let taskOne = "";
+
+for (let i = 10; i <= 20; i++) {
+    taskOne += i + ",";
+}
+
+console.log(taskOne.replace(/,\s*$/, ""));
+
+// Here's another variation using array
+
+const taskOneVariation = [];
+
+for (let i = 10; i <= 20; i++) {
+    taskOneVariation.push(i);
+}
+
+console.log(taskOneVariation.join(","));
+
+
+//2. Square numbers from 10 to 20
+
+const taskTwo = [];
+
+for (let i = 10; i <= 20; i++) {
+    const square = i * i;
+    taskTwo.push(square)
+}
+
+console.log(taskTwo.join(","))
```

Comment:
it will be easier to read

```suggestion
console.log(taskTwo.join(", "))
```